rs/ledger_suite/icrc1/index-ng/tests/common/mod.rs

```diff
@@ -34,6 +34,15 @@ const NAT_META_VALUE: u128 = u128::MAX;
 const INT_META_KEY: &str = "test:int";
 const INT_META_VALUE: i128 = i128::MIN;
 
+pub fn account(owner: u64, subaccount: u128) -> Account {
```

Comment:
Should this be added to the `Account` struct implementation?

Reply:
This function is only used for testing, so I wouldn't add it to the `Account` struct implementation. It could potentially be renamed to avoid confusion, but it's used in so many places so that to avoid polluting this PR, I'd do it in a separate ticket. In this PR I just moved it to the `common` module to be able to use it in the tests that I added in a separate module.